Flank/src/test/java/com/walmart/otto/filter/TestFiltersTest.java

```diff
@@ -0,0 +1,118 @@
+package com.walmart.otto.filter;
+
+import static com.walmart.otto.filter.TestFilters.notIgnored;
+import static com.walmart.otto.filter.TestMethodFixtures.BAR_CLASSNAME;
+import static com.walmart.otto.filter.TestMethodFixtures.BAR_PACKAGE;
+import static com.walmart.otto.filter.TestMethodFixtures.FOO_CLASSNAME;
+import static com.walmart.otto.filter.TestMethodFixtures.FOO_PACKAGE;
+import static com.walmart.otto.filter.TestMethodFixtures.WITHOUT_FOO_ANNOTATION;
+import static com.walmart.otto.filter.TestMethodFixtures.WITHOUT_IGNORE_ANNOTATION;
+import static com.walmart.otto.filter.TestMethodFixtures.WITH_FOO_ANNOTATION;
+import static com.walmart.otto.filter.TestMethodFixtures.WITH_FOO_ANNOTATION_AND_PACKAGE;
+import static com.walmart.otto.filter.TestMethodFixtures.WITH_IGNORE_ANNOTATION;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import com.walmart.otto.testsupport.TestUtils;
+import java.nio.file.Path;
+import org.junit.Test;
+
+public class TestFiltersTest {
```

Comment:
I would like to see a test with `package com.p1, package com.p2, class x`

Reply:
one more `package x; notPackage x`. What should happen?

Reply:
Will create a filter that return always false. When you run flank with such a filter it will complain because there are no tests to run.